lib/mayaUsd/ufe/UsdTransform3dMayaXformStack.cpp

```diff
@@ -466,6 +469,9 @@ Ufe::Vector3d UsdTransform3dMayaXformStack::rotation() const
     }
     UsdGeomXformOp r = getOp(NdxRotate);
     TF_AXIOM(r);
+    if (!r.GetAttr().HasValue()) {
```

Comment:
Here are my findings:
- UsdTransform3d.cpp: HasAttribute() checks in place.
- UsdTransform3dBase.cpp: default implementations only, no attribute access.
- UsdTransform3dCommonAPI.cpp: no direct attribute access, only accessing
      through UsdGeomXformCommonAPI object, which returns the identity matrix
      if no value has been authored, which is fine.
- UsdTransform3dFallbackMayaXformStack.cpp: no value attribute access.
- UsdTransform3dMatrixOp.cpp: getValue() utility is a problem, it ignores the
      UsdAttribute::Get() boolean return value, which may be false.  Entered as MAYA-110061, assigned to me.
 
      Otherwise, uses UsdGeomXformOp::GetOpTransform(), which returns the
      identity matrix if no value has been authored, which is fine.
- UsdTransform3dMayaXformStack.cpp: fixed in this pull request.
- UsdTransform3dSetObjectMatrix.cpp: no direct attribute access.